test/server/honeypot.test.ts

```diff
@@ -3,6 +3,10 @@ import CryptoJS from "crypto-js";
 
 import { Honeypot, SpamError } from "../../src";
 
+function invariant(condition: any, message: string): asserts condition {
+  if (!condition) throw new Error(message);
+}
```

Comment:
Added this because typescript's not super jazzed about the `validFromFieldName` possibly being `null`. The fact it's null wouldn't impact user code though so it's no big deal there. This just makes our own tests easier to work with.